packages/lesswrong/lib/eaGivingSeason.ts

```diff
@@ -23,7 +23,7 @@ export const votingThankYouImageId = "voting_portal_hero";
 
 const votingAccountCreationCutoff = new Date("2023/10/23");
 
-const userCanVoteInDonationElection = (
+export const userCanVoteInDonationElection = (
```

Comment:
Ah I hadn't see this existed, I was doing a similar thing in https://github.com/ForumMagnum/ForumMagnum/pull/8340 . I'll switch to using this in that brach